tests/components/remote_calendar/test_config_flow.py

```diff
@@ -34,54 +37,112 @@ async def test_form_import_ics(
             CONF_URL: CALENDER_URL,
         },
     )
+    assert result2["title"] == CALENDAR_NAME
+    assert result2["data"] == {
+        CONF_CALENDAR_NAME: CALENDAR_NAME,
+        CONF_URL: CALENDER_URL,
+    }
     assert result2["type"] is FlowResultType.CREATE_ENTRY
```

Comment:
Please put this first by convention and I think easier to understand error message if the test fails